projects/baby_driver/scripts/gpio_port.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for identifying GPIO ports."""
+
+import enum
+
+class GpioPort(enum.Enum):
+    """
+    An enumeration to specify GPIO ports by name (e.g. GpioPort.A instead of 0).
+    """
+    A = 0
+    B = 1
+    C = 2
+    D = 3
+    E = 4
+    F = 5
+    NUM_GPIO_PORTS = 6
```

Comment:
I know it's in the enum in C, but `NUM_GPIO_PORTS` should really be a top level constant on its own, not a member of the enum.